core/src/Instance/Authority.cs

```diff
@@ -38,7 +38,7 @@ namespace Microsoft.Identity.Core.Instance
     internal abstract class Authority
     {
         internal static readonly HashSet<string> TenantlessTenantNames =
-            new HashSet<string>(new[] {"common", "organizations"});
+            new HashSet<string>(new[] {"common", "organizations", "consumers"});
```

Comment:
thanks for adding this @SomkaPe. I guess consumer implies the MSA tenant Id, but I also guess we'd have it in the IAccount.